base/applications/rapps/include/appview.h

```diff
@@ -281,12 +281,12 @@ class CMainToolbar :
     const INT m_iToolbarHeight;
     DWORD m_dButtonsWidthMax;
 
-    WCHAR szInstallBtn[MAX_STR_LEN];
-    WCHAR szUninstallBtn[MAX_STR_LEN];
-    WCHAR szModifyBtn[MAX_STR_LEN];
-    WCHAR szSelectAll[MAX_STR_LEN];
-    WCHAR szRefreshBtn[MAX_STR_LEN];
-    WCHAR szUpdateDbBtn[MAX_STR_LEN];
+    ATL::CStringW szInstallBtn;
+    ATL::CStringW szUninstallBtn;
+    ATL::CStringW szModifyBtn;
+    ATL::CStringW szSelectAll;
+    ATL::CStringW szRefreshBtn;
+    ATL::CStringW szUpdateDbBtn;
```

Comment:
These don't need to be stored in the object, the toolbar will copy the strings.

Reply:
Moved.